.github/workflows/ci.yaml

```
@@ -725,15 +731,15 @@ jobs:
 
   miri-tests:
     name: "Miri (unsafe code verification)"
-    runs-on: blacksmith-4vcpu-ubuntu-2404
+    runs-on: ${{ (((github.event_name != 'pull_request' && github.event_name != 'pull_request_target') || github.event.pull_request.head.repo.full_name == github.repository) && fromJSON('["self-hosted", "ix"]')) || 'blacksmith-4vcpu-ubuntu-2404' }}
     needs: determine_changes
     # TEMPORARILY DISABLED: miri tests timing out at 25 minutes, blocking CI/CD
     # TODO: Re-enable once miri test performance is addressed
     if: false # needs.determine_changes.outputs.unsafe == 'true'
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)